sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo.csproj

```diff
@@ -12,7 +12,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Azure.Monitor.OpenTelemetry.LiveMetrics\src\Azure.Monitor.OpenTelemetry.LiveMetrics.csproj" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" />
```

Comment:
nit: this reference shouldn't be needed. I would expect this to come transitively from the Exporter project reference.

Reply:
please ignore